lld/wasm/Driver.cpp

```diff
@@ -797,7 +821,7 @@ static void writeWhyExtract() {
 // Equivalent of demote demoteSharedAndLazySymbols() in the ELF linker
 static void demoteLazySymbols() {
   for (Symbol *sym : symtab->symbols()) {
-    if (auto* s = dyn_cast<LazySymbol>(sym)) {
+    if (auto *s = dyn_cast<LazySymbol>(sym)) {
```

Comment:
Can you revert these hunks that are formatting only (I find it detracts from the substance of a change to mix these in with semantic changes)?

(To format just the parts of the file you touch you can use `git clang-format origin/main`)

Reply:
For some reason, `git clang-format origin/main` did not do anything for me on Windows. I force pushed and manually removed the non-related formatted changes.

Reply:
For some reason, git clang-format origin/main did not do anything for me on Windows. I force pushed and manually removed the non-related formatted changes.

Reply:
Yes, sorry, that command would have prevented the whole-file-formatting issue, it won't undo it once its done.